FlashLFQ/Identification.cs

```diff
@@ -33,6 +33,25 @@ public class Identification
             UseForProteinQuant = useForProteinQuant;
             PosteriorErrorProbability = posteriorErrorProbability;
         }
+        /// <summary>
+        /// This Identification object was created for instances when the modified sequence isn't parsable into a formula
+        /// for calcuation of the peptide/proteoform mass. 
+        /// </summary>
+        public Identification(SpectraFileInfo fileInfo, string ModifiedSequence, double monoisotopicMass, 
```

Comment:
don't need this second `Identification` constructor; you can use the original constructor and simply pass in `null` for the `BaseSequence` parameter